data/todo.py

```
@@ -1,24 +1,66 @@
+        response = (
+            supabase.table("todo").select("id", count='exact')
+            .eq("user_id", user_id)
+            .gte("created_at", today.isoformat())
+            .lt("created_at", tomorrow.isoformat())
             .execute()
         )
-        return resp.data
\ No newline at end of file
+        
+        print(f"DEBUG: Supabase count response: {response.count}")
+        return response.count
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)